pytest-selenium/tests/test_toc.py

```diff
@@ -64,3 +64,34 @@ def test_toc_toggle_button_opens_and_closes(selenium, base_url, book_slug, page_
     else:
 
         pytest.fail("window must be either mobile or desktop size")
+
+
+@markers.test_case("C476818")
+@markers.parametrize("book_slug,page_slug", [("college-physics", "1-1-physics-an-introduction")])
+@markers.nondestructive
+@markers.mobile_only
+def test_toc_disables_interacting_with_content_on_mobile(selenium, base_url, book_slug, page_slug):
+
+    # GIVEN: A page URL in the format of {base_url}/books/{book_slug}/pages/{page_slug}
+    # AND: A mobile resolution
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    toolbar = content.toolbar
+    attribution = content.attribution
+    sidebar = content.sidebar
+
+    # WHEN: the toc is open
+    toolbar.click_toc_toggle_button()
+
+    # THEN: The links in the content should be disabled
+    content.assert_element_not_interactable_exception(content.next_link)
+    content.assert_element_not_interactable_exception(content.previous_link)
+    content.assert_element_not_interactable_exception(attribution.attribution_link)
+
+    # AND scrolling over it should do nothing
+    assert content.is_scroll_locked
+
+    # AND clicking anywhere in the content overlay should just close the TOC and content stays in the same page
+    content.click_content_overlay()
+    assert not sidebar.header.is_displayed
+    initial_url = base_url + "/books/" + book_slug + "/pages/" + page_slug
```

Comment:
instead of calculating this url, can you save `selenium.current_url` to a variable before you start clicking on stuff?